A-set/667A. Pouring Rain.py

```diff
@@ -0,0 +1,9 @@
+import math
+d,h,v,e = map(int, input().split())

```

Comment:
Please add a new line after import. when you are declaring the variables, it's a good practice to use a space after the comma. Like, `d, h, v, e`